app/Models/CenterProject.php

```diff
@@ -60,4 +60,41 @@ public function levels()
         // dump($q->toSql(), $q->getBindings());
         return $q;
     }
-}
+
+    public function getAssign($center_id, $project_id, $year)
```

Comment:
Give this function a better name? getAssignment? getAssignedProject? Not exactly sure what this function does.